forest/src/daemon.rs

```diff
@@ -40,7 +42,21 @@ pub(super) async fn start(config: Config) {
         });
 
     // Initialize keystore
-    let mut ks = PersistentKeyStore::new(config.data_dir.to_string()).unwrap();
+    let passphrase = if let true = config.encrypt_keystore {
+        print!("Keystore passphrase: ");
+        std::io::stdout().flush().unwrap();
+        Some(read_password().expect("Error reading passphrase"))
+    } else {
+        None
+    };
+
+    let mut ks = PersistentKeyStore::new(
```

Comment:
Just as a reader it would be nice to know why we expect this to never fail. Either a comment or switching to `.expect("reason")` would be nice

Reply:
If the keystore fails to be created, its either because of configuration issues, or creating the necessary files. The errors were propagated up in the associated function. Changed this to use `.expect()`.